### DISCUSSION TITLE
How do I actually get cursor position in the new versions?

### DISCUSSION THREAD

**Discussion by velllu** [bevyengine/bevy#8773] [Q&A]
Every way I've seen was for an old version of bevy and it now doesn't work anymore. What is the current best way?

**Comment by hymm** [comment]
you want the event `CursorMoved`.

https://github.com/bevyengine/bevy/blob/1c5c94715cb17cda5ae209eef12a938501de90b5/examples/input/mouse_input_events.rs#L19
https://docs.rs/bevy/latest/bevy/prelude/struct.CursorMoved.html
